Library/Koala/Koala.swift

```diff
@@ -1824,6 +1824,35 @@ public final class Koala {
     }
   }
 
+  public func trackChangedCurrency(_ currency: Currency) {
```

Comment:
Just an idea, but this `Koala.swift` file is a 2000 line beast of a file. Should we consider breaking it up into extensions by feature?

Reply:
Yeah, I like this idea! I would do it in other PR, though, but would be happy to refactor `Koala.swift`

Reply:
I've already added some pragma marks to `Koala.swift` for better navigation so it should be slightly easier (those changes are on `master` branch